libs/design-system/filter-container/src/lib/filter-container.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="filter-container">
+  <div class="top-row">
+    @if (tooltip()) {
```

Comment:
Bind the result using `as`, i.e. `@if (tooltip(); as tooltipData)` and use `tooltipData` instead of `tooltip()!`